og_access/og_access.test

```diff
@@ -344,6 +344,32 @@ class OgAccessTestCase extends DrupalWebTestCase {
     drupal_static_reset('node_access');
 
     $this->assertFalse(node_access('view', $node, $user2), 'Group member can not view unpublished node.');
+
+    $groups1 = og_get_entity_groups('user', $user2, array(OG_STATE_ACTIVE), NULL, TRUE);
+    $groups2 = og_get_groups_by_user($user2, 'node', TRUE);
+    $groups1 = $groups1 ? $groups1 : array('node' => array());
```

Comment:
I think it would be more readable to have the assert deal with the raw result, instead of adding a default value.